hexrd/ui/rerun_clustering_dialog.py

```diff
@@ -0,0 +1,97 @@
+import copy
+import numpy as np
+from pathlib import Path
+
+from hexrd.ui.async_worker import AsyncWorker
+from hexrd.ui.hexrd_config import HexrdConfig
+from hexrd.ui.ui_loader import UiLoader
+
+from PySide2.QtWidgets import QDialog, QFileDialog
+from PySide2.QtCore import Qt
+
+
+class RerunClusteringDialog(QDialog):
+
+    def __init__(self, indexing_runner, parent=None):
+        super().__init__(parent)
+
+        loader = UiLoader()
+        self.ui = loader.load_file('rerun_clustering_dialog.ui', parent)
+        self.indexing_runner = indexing_runner
+        self.qfib = None
+        self.completeness = None
+        self.setup_gui()
+        self.setup_connections()
+
+    def setup_gui(self):
+        idx_cfg = HexrdConfig().indexing_config['find_orientations']
+        clustering_data = idx_cfg['clustering']
+        self.ui.radius.setValue(clustering_data['radius'])
+        self.ui.completeness.setValue(clustering_data['completeness'])
+        self.ui.algorithms.setCurrentText(clustering_data['algorithm'])
+        needs_min_samples = self.indexing_runner.clustering_needs_min_samples
+        self.ui.min_samples_label.setVisible(needs_min_samples)
+        self.ui.min_samples.setVisible(needs_min_samples)
+        if needs_min_samples:
+            self.ui.min_samples.setValue(self.indexing_runner.min_samples)
+
+    def setup_connections(self):
+        self.ui.load_file.clicked.connect(self.load_file)
+        self.ui.buttonBox.accepted.connect(self.accept)
```

Comment:
This connection is already made in the UI file, and doesn't need to be made here.

Also, it might be nice if `buttonBox` is renamed to `button_box`, just to be consistent with snake_case.

Reply:
I'm not sure what I'm missing here because it is certainly already set in the UI file, but removing the connection here results in the `accept` function not being called. Updated to use snake_case though.

Reply:
Hmm, okay. We'll keep it as-is and hopefully figure out why later.